kubetools/kubernetes/config/__init__.py

```diff
@@ -35,8 +35,19 @@ def _ensure_image(
     container, context_name_to_image,
     deployment_name=None,
     container_name=None,
+    default_registry=None,
 ):
     if 'image' in container:
+        registry_image = container['image'].split('/')
+        if len(registry_image) == 1:
+            # If the image is commonly available from DockerHub
+            if default_registry is None:
+                container['image'] = registry_image[0]
+
+            # If registry not specified in the kubetools.yml file
+            else:
+                image = container['image']
+                container['image'] = f'{default_registry}/{image}'
```

Comment:
```suggestion
        if len(registry_image) == 1 and default_registry is not None:
            # If registry is not specified but a default was provided
            container['image'] = f'{default_registry}/{container['image']}'
```

The `if` part of the `if..else` was a no-op

Reply:
https://github.com/EDITD/kubetools/pull/141/commits/c7856312ccb204374777d69ef580e151d7be594c